$NetBSD: patch-tclpkg_gv_Makefile.am,v 1.2 2022/07/22 12:04:46 micha Exp $

Add missing -I flags to point to perl files.

See https://gitlab.com/graphviz/graphviz/issues/1576.
Issue was closed, but patch is still missing. Regression?

https://gitlab.com/graphviz/graphviz/-/merge_requests/2758

--- tclpkg/gv/Makefile.am.orig	2021-10-16 16:39:12.000000000 +0000
+++ tclpkg/gv/Makefile.am
@@ -180,7 +180,7 @@ nodist_libgv_perl_la_SOURCES = gv_perl.c
 libgv_perl_la_SOURCES = $(BASESOURCES) gv_dummy_init.c
 libgv_perl_la_LIBADD = $(BASELIBS) $(PERL_LIBS)
 libgv_perl_la_LDFLAGS = -module -avoid-version $(PERL_LD)
-libgv_perl_la_CPPFLAGS = $(BASECPPFLAGS) $(PERL_CC)
+libgv_perl_la_CPPFLAGS = $(BASECPPFLAGS) $(PERL_INCLUDES) $(PERL_CC)
 $(PERL_data): gv_perl.cpp
 gv_perl.cpp: gv.i
 	$(SWIG) -c++ $(SWIG_PERL_OPT) -o $@ $(srcdir)/gv.i
